Common/Helpers/ProcessHelper.cs

```diff
@@ -846,13 +846,22 @@ private enum ShowWindowEnum
             Restore = 9, ShowDefault = 10, ForceMinimized = 11
         };
 
-        /**
-         * Finds the process by name and sets the main window to the foreground.
-         */
-        public static void RestoreProcessWindowState(string processName)
+        /// <summary>
+        /// WFN process names.
+        /// </summary>
+        public enum WFNProcessEnum

```

Comment:
Using constants would be better in fact. Enums are not meant to be used this way :-)

Reply:
see above.